tools/jdbc/src/jni/functions.cpp

```diff
@@ -0,0 +1,353 @@
+// This file was generated by tools/jdbc/generator.py
```

Comment:
What is the `h2w` invocation that triggers this? @Mause

Reply:
```sh
python3 generator.py
```
?

Reply:
Hmmm. Thank you for confirming.

When I tried running it before I've got many `header2whathever` related errors and suspected the script wasn't supposed to be called directly, but it works now. I think I was calling it from the wrong path.